packages/client/screens/UserProfile/UserFollowingSearch/__test__/UserFollowingSearch.test.tsx

```diff
@@ -0,0 +1,723 @@
+import {
```

Comment:
See comments for `packages/client/screens/UserProfile/UserFollowersSearch/__tests__/UserFollowersSearch.test.tsx`.

Reply:
?